package.json

```diff
@@ -37,12 +37,14 @@
     "coveralls": "3.0.2",
     "css-loader": "1.0.0",
     "d3": "5.7.0",
+    "istanbul-instrumenter-loader": "^3.0.1",
     "jasmine": "2.6.0",
     "jasmine-jquery": "2.1.1",
     "jquery": "3.2.1",
     "karma": "2.0.4",
     "karma-chrome-launcher": "2.2.0",
     "karma-coverage": "1.1.2",
+    "karma-coverage-istanbul-reporter": "^2.0.4",
```

Comment:
Are old package still in use? Otherwise, let remove useless packages from dependencies

Reply:
Done!